docs/docs/reference/Query_Syntax.md

```diff
@@ -215,11 +218,16 @@ The syntax is `(foo bar) => { $attribute: value; $attribute:value; ...}`, e.g:
 
 The supported attributes are:
 
-* **$weight**: determines the weight of the sub-query or token in the overall ranking on the result (default: 1.0).
+1. **$weight**: determines the weight of the sub-query or token in the overall ranking on the result (default: 1.0).
 2. **$slop**: determines the maximum allowed "slop" (space between terms) in the query clause (default: 0).
 3. **$inorder**: whether or not the terms in a query clause must appear in the same order as in the query, usually set alongside with `$slop` (default: false).
 4. **$phonetic**: whether or not to perform phonetic matching (default: true). Note: setting this attribute on for fields which were not creates as `PHONETIC` will produce an error.
```

Comment:
Please check the note sentence. It's unclear.

Reply:
I agree - but it's out of this PR scope... We can open an issue for that